components/prism-keyman.js

```diff
@@ -0,0 +1,14 @@
+Prism.languages.keyman = {
+	'comment': /\b[cC]\s.*?(\r?\n|$)/,
+	'function': /\[\s*((CTRL|SHIFT|ALT|LCTRL|RCTRL|LALT|RALT)\s+)*(([T|K|U]_[a-z0-9_]+)|(".+")|('.+'))\s*\]/i,  // virtual key
```

Comment:
T, U are referenced in http://help.keyman.com/developer/9.0/guides/touch-layouts/creating-a-touch-keyboard-layout-for-amharic-the-nitty-gritty.php. 

Also will tweak quoted strings (non-greedy match).

Aware that ordering virtual keys ahead of strings will format "[K_A]" wrongly but in practice this would never be used so not worth supporting.